enzyme/benchmarks/ReverseMode/ba/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # Run regression and unit tests
-add_lit_testsuite(bench-matdescent "Running enzyme benchmarks tests"
+add_lit_testsuite(bench-ba-reverse "Running enzyme benchmarks tests"
```

Comment:
It still seems like the git move was messed up?

Reply:
its gonna be messed up again if we squash and merge this PR. Maybe we can merge it without squashing this time...

Reply:
Sure, though this particular file seems weird since its a move from matdescent to ba

Reply:
Oh, huh it looks like the commit itself is fine